### ISSUE TITLE
Measure-FleetCoreWorkload - Import-Counter function

### ISSUE THREAD

**Issue by jayr0x** [microsoft/diskspd#191]
Hello guys,
for context, I had to set my locale OS to English US to be able to run the script. After the first run, I've encountered the error below. 
![image](https://user-images.githubusercontent.com/117855595/201159169-d159a045-8b68-4fae-87f9-294a7d4aaae8.png)

in the coreworkload log : 
-11-10T20:18:31: COMPLETION CHECK 7 : 64/64 done (0,02s, total 12,59s)
PS>TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
>> TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
>> TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
>> TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
>> TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
2022-11-10T20:18:53: Iteration file content saved to C:\ClusterStorage\collect\result\error for triage
PS>TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
>> TerminatingError(Import-Counter): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: L’objet spécifié n’a pas été trouvé sur l’ordinateur."
**********************
Windows PowerShell transcript end
End time: 20221110201853
**********************
Do you have any idea what could be missing ? 
The error says : the specified object can't be found on the computer. 
My guess would be a specific counter name can't be found ?

**Comment by dl2n** [comment]
Could you look at the output of "typeperf -q" and/or tell me if the performance counter set names are localized when in the French fr-fr locale? (I assume that was your original)

**Comment by jayr0x** [comment]
since i switch to english, it is correctly localized. 
[typeperf.txt](https://github.com/microsoft/diskspd/files/9994690/typeperf.txt)

**Comment by jayr0x** [comment]
any update possible on this "issue" ?

**Comment by dl2n** [comment]
Sorry, just to be clear: when the localization is fr-fr (or not en-us), are the names of the counter sets (the output of typeperf -q) localized? That is what I was looking for when asking for that output - in the fr-fr localization.

Your txt output shows what I am already familiar with in the en-us case.

**Comment by jayr0x** [comment]
Yes like I said above, they are localized when the OS is in french version. But I don't think that matters because I've switched to EN-US when running the script. 
[typeperf_FR.txt](https://github.com/microsoft/diskspd/files/10112418/typeperf_FR.txt)

**Comment by KaiKirste** [comment]
Good news (for me)!

It works after i have setup my nodes and cluster from scratch with English Windows Server 2019 Core Setup. 
Prio test with German GUI Setup including change of systemlanguage to english and complete Setup of English GUI doesn't work.


Greetings
